cfme/services/catalogs/orchestration_template.py

```diff
@@ -140,8 +140,10 @@ def create(self, content):
         view = navigate_to(self, "AddTemplate")
         if(self.template_type == "CloudFormation Templates"):
             temp_type = "Amazon CloudFormation"
-        else:
+        elif(self.template_type == "Heat Templates"):
```

Comment:
maybe
        else:
            raise Exception("ERROR template_type needs to be one of the above")